README.md

```diff
@@ -73,15 +73,36 @@
 $ npm install --save project-version
 ```
 
-## Usage
-
+## JS Usage
 ```js
 const version = require('project-version');
 
+// Caution: console.log outputs newlines, use process.stdout.write to avoid whitespaces
 console.log(version);
 // =>  '1.0.1'
 ```
 
+## CLI Usage
```

Comment:
Call that CLI
Have a look to this example:
https://github.com/simonepri/osm-geojson#cli